Pcos Detection/Model/README.md

```diff
@@ -0,0 +1,67 @@
+**Pcos Detection**
+
+🎯**GOAL**
+
+The aim of the project is to build a model which would predict whether a person has the Pcos Detection
+
+🧵**DATASET**
+
+The dataset was taken from the Kaggle.
+
+🧾**DESCRIPTION**
+
+In this project, we compare the accuracies of different models to predict the pcos in the person.
+
+🧮**WHAT I HAD DONE**
+
+Steps followed:
+* Downloaded the dataset from Kaggle.
+* Checked for null values and performed exploratory data analysis
+* Built models using the following algorithms: Logistic Regression, KNN, Random Forest
+* Compared the accuracies
+* Then performed cross validation on all the algorithms to see if the accuraries imrpove.
+
+🚀**Models Implemented**
+
+1) Logistic Regression
+2) Decision Tree
+3) Random forest
+4) Gradient Boosting
+5) Adaboost
+6) Naive Bayes
+7) Support Vector Machine
+8) K-Nearest Neighbors
+
+📚**LIBRARIES NEEDED**
+
+* Pandas
+* Matplotlib
+* Seaborn
+* Numpy
+* Sklearn
+
+📊**Exploratory Data Analysis Results**
+<a href="C:\Users\KIIT\Desktop\SSOC'24\ML-Crate\Pcos Detection\Images\Screenshot 2024-06-07 114101.png">Confusion Matrix</a>
+<a href="C:\Users\KIIT\Desktop\SSOC'24\ML-Crate\Pcos Detection\Images\Screenshot 2024-06-07 114118.png">Class Dostribution using SMOTE</a>
+
+
+📈 **Performance of the Models based on the Accuracy Scores**
```

Comment:
The markdown script you have written is not creating a table, instead it is showing one after another.

This is what you need to fix.